platform_secrets/service.py

```diff
@@ -103,23 +182,23 @@ async def remove_secret(self, secret: Secret) -> None:
             secret_name = self._get_kube_secret_name(
                 secret.project_name, secret.org_name
             )
-            await self._kube_client.remove_secret_key(secret_name, secret.key)
+            await self._kube_api.remove_secret_key(
+                secret_name, secret.key, namespace_name=secret.namespace_name
+            )
         except (ResourceNotFound, ResourceInvalid):
             raise SecretNotFound.create(secret.key)
 
     async def get_all_secrets(
         self,
+        org_name: str,
+        project_name: str,
         with_values: bool = False,
-        org_name: Optional[str] = None,
-        project_name: Optional[str] = None,
     ) -> list[Secret]:
-        label_selectors = []
-        if org_name and org_name.upper() == NO_ORG:
-            label_selectors += [f"!{SECRET_API_ORG_LABEL}"]
-        elif org_name:
-            label_selectors += [f"{SECRET_API_ORG_LABEL}={org_name}"]
+        label_selectors = [f"{SECRET_API_ORG_LABEL}={org_name}"]
```

Comment:
I think we will loose access to existing no org secrets, currently they don't have SECRET_API_ORG_LABEL

Reply:
@zubenkoivan I though that during the migration, if secret doesn't belong to an org, we will populate a label for it, e.g. `platform.neuromation.io/secret-api-org-name=no-org`. 
I think this might simplify the code for us. WDYT?

Reply:
sounds good